docs/administration/back_office/customize_search_suggestion.md

```diff
@@ -0,0 +1,157 @@
+---
+description: Customize search suggestion configuration and sources.
+---
+
+# Customize search suggestion
+
+In the Back Office, when you start typing in the search field on the top bar, suggestions about what you could be looking for show up directly under the field.
+For more information about using this feature to search for content, see [user documentation]([[= user_doc =]]/search/search_for_content).
+
+## Configuration
+
+By default, suggestions start showing up after the user types in at least 3 characters, and 5 suggestions are presented.
+This can be changed with [SiteAccess aware dynamic configuration](dynamic_configuration.md), by setting the following [scoped](multisite_configuration.md#scope) parameters:
```

Comment:
```suggestion
By default, suggestions start showing up after the user types in at least 3 characters, and 5 suggestions are presented.
This can be changed with the following [scoped](multisite_configuration.md#scope) configuration:
```

Reply:
Having https://doc.ibexa.co/en/latest/administration/configuration/dynamic_configuration/ and https://doc.ibexa.co/en/latest/multisite/multisite_configuration/#scope side by side was disturbing…